src/test/java/org/prebid/server/bidder/goldbach/GoldbachBidderTest.java

```diff
@@ -0,0 +1,539 @@
+        final Bid bid = Bid.builder().id("testBidId").ext(bidExt).build();
+        final BidderCall<BidRequest> httpCall = givenHttpCall(bid);
+
+        // when
+        final Result<List<BidderBid>> result = target.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).containsExactly(
+                BidderBid.of(
+                        bid,
+                        BidType.banner,
+                        "USD"
+                )
+        );
+    }
+
+    @Test
+    public void makeBidsShouldReturnValidBidsAndErrorsIfThereAreBothValidAndInvalidBidsInInput() {
+        // given
+        final ObjectNode validBidExt = mapper.createObjectNode()
+                .set("prebid", mapper.createObjectNode().put("type", "banner"));
+        final ObjectNode invalidBidExt = mapper.createObjectNode()
+                .set("prebid", mapper.createObjectNode());
+        final Bid validBid = Bid.builder().id("validBidId").ext(validBidExt).build();
+        final Bid invalidBid = Bid.builder().id("invalidBidId").ext(invalidBidExt).build();
+        final BidderCall<BidRequest> httpCall = givenHttpCall(validBid, invalidBid);
+
+        // when
+        final Result<List<BidderBid>> result = target.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getErrors()).containsExactly(BidderError.badInput("No media type for bid invalidBidId"));
+        assertThat(result.getValue()).containsExactly(BidderBid.of(validBid, BidType.banner, "USD"));
+    }
+
+    private static BidRequest givenBidRequest() {
+        return givenBidRequest(identity());
+    }
+
+    private static BidRequest givenBidRequest(Imp... imps) {
+        return givenBidRequest(identity(), imps);
+    }
+
+    private static BidRequest givenBidRequest(UnaryOperator<BidRequest.BidRequestBuilder> bidRequestCustomizer,
+                                              Imp... imps) {
+
+        return bidRequestCustomizer
+                .apply(BidRequest.builder()
+                        .id("testBidRequestId")
+                        .imp(List.of(imps)))
+                .build();
+    }
+
+    private static Imp givenImp() {
+        return givenImp(givenImpExt());
+    }
+
+    private static Imp givenImp(ObjectNode impExt) {
+        return givenImp(null, impExt);
+    }
+
+    private static Imp givenImp(String impId, ObjectNode impExt) {
+        return Imp.builder()
+                .id(impId)
+                .ext(impExt)
+                .build();
+    }
+
+    private static ObjectNode givenImpExt() {
+        return givenImpExt("testPublisherId", "testSlotId", null);
+    }
+
+    private static ObjectNode givenImpExt(String publisherId, String slotId) {
+        return givenImpExt(publisherId, slotId, null);
+    }
+
+    private static ObjectNode givenImpExt(String publisherId, String
+            slotId, Map<String, List<String>> customTargeting) {
```

Comment:
Don't tear up the declaration like this. Either everything is on one line, or each parameter is on a new line.